test/controllers/preview_controller_test.exs

```diff
@@ -44,9 +25,7 @@ defmodule CodeCorps.PreviewControllerTest do
     end
```

Comment:
Isn't it testing that we get a 201 response when creating a preview?  This test was in the comments controller as well.